js/jquery.keyboard.js

```diff
@@ -2072,30 +2072,34 @@ http://www.opensource.org/licenses/mit-license.php
 
 		data.html = '<span class="' + kbcss.keyText + '">' +
 			// this prevents HTML from being added to the key
-			keys.name.replace(/[\u00A0-\u9999]/gim, function (i) {
+			keys.name.replace(/&/g,"&amp;").replace(/</g,"&lt;").replace(/>/g,"&gt;").replace(/[\u00A0-\u9999]/gim, function (i) {
 				return '&#' + i.charCodeAt(0) + ';';
 			}) +
 			'</span>';
 
-		data.$key = base.keyBtn
-			.clone()
-			.attr({
-				'data-value': regKey ? keys.name : keys.action, // value
-				'data-name': keys.action,
-				'data-pos': base.temp[1] + ',' + base.temp[2],
-				'data-action': keys.action,
-				'data-html': data.html
-			})
-			// add 'ui-keyboard-' + data.name for all keys
-			//  (e.g. 'Bksp' will have 'ui-keyboard-bskp' class)
-			// any non-alphanumeric characters will be replaced with
-			//  their decimal unicode value
-			//  (e.g. '~' is a regular key, class = 'ui-keyboard-126'
-			//  (126 is the unicode decimal value - same as &#126;)
-			//  See https://en.wikipedia.org/wiki/List_of_Unicode_characters#Control_codes
-			.addClass(keyClass)
-			.html(data.html)
-			.appendTo(base.temp[0]);
+		try{
+			data.$key = base.keyBtn
+				.clone()
+				.attr({
+					'data-value': regKey ? keys.name : keys.action, // value
+					'data-name': keys.action,
+					'data-pos': base.temp[1] + ',' + base.temp[2],
+					'data-action': keys.action,
+					'data-html': data.html
+				})
+				// add 'ui-keyboard-' + data.name for all keys
+				//  (e.g. 'Bksp' will have 'ui-keyboard-bskp' class)
+				// any non-alphanumeric characters will be replaced with
+				//  their decimal unicode value
+				//  (e.g. '~' is a regular key, class = 'ui-keyboard-126'
+				//  (126 is the unicode decimal value - same as &#126;)
+				//  See https://en.wikipedia.org/wiki/List_of_Unicode_characters#Control_codes
+				.addClass(keyClass)
+				.html(data.html)
+				.appendTo(base.temp[0]);
+		}catch(e){
+			console.log(data.html);
+		}
```

Comment:
Why did this need to be wrapped in a try catch function?

Reply:
Sorry its not necessary. it was a help to fin out why the keyboard don't work